helm/memcached-operator/deploy/role.yaml

```diff
@@ -18,22 +18,21 @@ rules:
   verbs:
   - '*'
 - apiGroups:
-  - ""
+  - policy
   resources:
-  - services
+  - poddisruptionbudgets
   verbs:
   - '*'
 - apiGroups:
-  - apps
+  - ""
   resources:
-  - deployments
-  - statefulsets
+  - services
   verbs:
   - '*'
 - apiGroups:
-  - policy
+  - apps
   resources:
-  - poddisruptionbudgets
+  - statefulsets
   verbs:
   - '*'
 - apiGroups:
```

Comment:
It is the role file generated by the new version whiteout manual updates.
Note that all worked fine 👍